### PR TITLE
Lms 123

### PR DESCRIPTION
**Pull request by DemchyshynV**

### CODE REVIEW

File: src/database/models/user.schema.ts [hunk 1]
```diff
@@ -21,12 +21,12 @@ UserSchema = new Schema({
     },
     password: {
         type: String,
-        required: true
+        required: true,
+        select: false
```
Comment (VictorKmin) [inline src/database/models/user.schema.ts]: Did you get this value when checking users's password?

File: src/middleware/group/isGroupPresent.middleware.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import { IGroup, IRequestExtended } from '../../interfaces';
+import { groupService } from '../../services/group';
+
+export const isGroupPresent = async (req: IRequestExtended, res: Response, next: NextFunction) => {
+    try {
+        const { group_id } = req.params;
+        const group = await groupService.getById(group_id) as IGroup;
+
+        if (!group) {
+            next(new ErrorHandler(
```
Comment (VictorKmin) [inline src/middleware/group/isGroupPresent.middleware.ts]: return next(

File: src/services/user/user.service.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ class UserService {
 
     getUserByParams(params: Partial<IUser>): Promise<any> {
         const UserModel = model<UserType>(DatabaseTablesEnum.USER_COLLECTION_NAME, UserSchema);
-        return UserModel.findOne(params) as any;
+        return UserModel.findOne(params).select('+password') as any;
```
Comment (VictorKmin) [inline src/services/user/user.service.ts]: For what you using `+password`

File: src/middleware/group/updateDateOfGroup.middleware.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { NextFunction, Response } from 'express';
+
+import { IGroupSubject, IRequestExtended } from '../../interfaces';
+import { groupService } from '../../services/group';
+
+export const updateGroupDate = async (req: IRequestExtended, res: Response, next: NextFunction) => {
+    try {
+        const group = req.group as IGroupSubject;
+        group.updated_at = new Date().toString();
```
Comment (VictorKmin) [inline src/middleware/group/updateDateOfGroup.middleware.ts]: Maybe better to update this info in updateGroup method in service?
Something like 
updateObject = { ...updateObject, updated_at = new Date().toString() }

File: src/database/models/user.schema.ts (hunk 1 above)
Comment (DemchyshynV) [reply src/database/models/user.schema.ts]: Yes, when I need this value I use "select ('+password)"

File: src/services/user/user.service.ts (hunk 3 above)
Comment (DemchyshynV) [reply src/services/user/user.service.ts]: for authController

File: src/middleware/group/updateDateOfGroup.middleware.ts (hunk 4 above)
Comment (DemchyshynV) [reply src/middleware/group/updateDateOfGroup.middleware.ts]: you are right

**Pull request merged**